runtime/src/lib.rs

```diff
@@ -309,13 +307,13 @@ impl pallet_ethereum::Config for Runtime {
 }
 
 parameter_types! {
-	pub const BlocksPerRound: u32 = 5;
+	pub const BlocksPerRound: u32 = 20;
 	pub const BondDuration: u32 = 2;
-	pub const MaxValidators: u32 = 5;
+	pub const MaxValidators: u32 = 16;
```

Comment:
I don't have any preferences on the number @crystalin

Is changing the config parameters an easy runtime grade?

Reply:
@JoshOrndorff tell me if I'm wrong but I thought it requires 100_000 token to join, and the reward is 100, so you need 1000 round to have enough.

Let's change it to 8, so that it is enough to add more and it will also force us to have a strategy to nominate different validator once we have 16. We will do a runtime upgrade to change it if needed

Reply:
Sorry, I forgot we changed the Issuance to 10_000.
@4meta5 can you set the issuance to 49 (it is per round right?) instead